components/global/Header/_navigation.scss

```diff
@@ -5,18 +5,11 @@ $_breakpoint: functions.header(layout-breakpoint);
 $_link-gap: 14px;
 
 .c-nav-list {
-  @include base.fluid-scale(
-    nav-width,
-    functions.header(nav-width-desktop),
-    functions.header(nav-width-mobile),
-    functions.break(tablet),
-    functions.break(mobile)
-  );
```

Comment:
width got more finicky with more levels so having this as a css var became more of a hindrance than a help